basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java

```diff
@@ -15,29 +13,22 @@
  */
 
 @Component
-public class HandlerFactory {
+public class HandlerFactory<T extends Handler> {
```

Comment:
@Alexey007gold Where is interface for factory ? We must inject interfaces, not implementation.